Manager/InMemory/AuthorizationCodeManager.php

```diff
@@ -36,4 +36,15 @@ public function clearExpired(): int
 
         return $count - \count($this->authorizationCodes);
     }
+
+    public function clearRevoked(): int
+    {
+        $count = \count($this->authorizationCodes);
+
+        $this->accessTokens = array_filter($this->authorizationCodes, static function (AuthorizationCode $accessToken): bool {
```

Comment:
@franjo-zadelj-trikoder `$this->accessTokens` and `AuthorizationCode $accessToken` ?